userspace/libsinsp/parsers.cpp

```diff
@@ -6025,3 +6028,52 @@ void sinsp_parser::free_event_buffer(uint8_t *ptr)
 		free(ptr);
 	}
 }
+
+void sinsp_parser::parse_memfd_create_exit(sinsp_evt *evt, scap_fd_type type)
+{
+	sinsp_evt_param* parinfo;
+	int64_t fd;
+	char *name;
+	uint32_t flags;
+	sinsp_fdinfo_t fdi;
+
+
+	ASSERT(evt->m_tinfo)
+	if(evt->m_tinfo == nullptr)
+	{
+		return;
+	}
+
+	/* ret (fd) */
+	parinfo = evt->get_param(0);
+	ASSERT(parinfo->m_len == sizeof(int64_t));
+	ASSERT(evt->get_param_info(0)->type == PT_FD);
+	fd = *(int64_t *)parinfo->m_val;
+	
+	/* name */
+	parinfo = evt->get_param(1);
```

Comment:
Suggesting to add a comment here something in line of:

Suppose you create a memfd name `libstest` the name will be `libstest` while on disk (e.g.   ` ls -l /proc/$PID/fd/$FD_NUM`) it may look like `/memfd:libstest (deleted)`

Reply:
Yes, this makes sense!!